Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettings.h

```diff
@@ -40,43 +75,69 @@ namespace AzToolsFramework
         PaintBrushSettings() = default;
         ~PaintBrushSettings() = default;
 
-        float GetRadius() const
-        {
-            return m_radius;
-        }
-        float GetIntensity() const
+        // Stroke settings
+
+        float GetIntensityPercent() const
         {
-            return m_intensity;
+            return m_intensityPercent;
         }
-        float GetOpacity() const
+        float GetOpacityPercent() const
         {
-            return m_opacity;
+            return m_opacityPercent;
         }
 
         PaintBrushBlendMode GetBlendMode() const
         {
             return m_blendMode;
         }
 
-        void SetRadius(float radius);
-        void SetIntensity(float intensity);
-        void SetOpacity(float opacity);
+        void SetIntensityPercent(float intensityPercent);
+        void SetOpacityPercent(float opacityPercent);
         void SetBlendMode(PaintBrushBlendMode blendMode);
 
+        // Stamp settings
+
+        float GetSize() const
+        {
+            return m_size;
+        }
```

Comment:
I'll skip this suggestion for now. I'm deliberately starting to push all mention of "radius" down into the implementation instead of the API at this point in the hopes that we'll get around to adding non-circular stamps at some point.